src/bash/Makefile

```diff
@@ -9,8 +9,17 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 
 	dget -u https://launchpad.net/debian/+archive/primary/+sourcefiles/bash/$(BASH_VERSION_FULL)/bash_$(BASH_VERSION_FULL).dsc
 
+
 	pushd bash-$(BASH_VERSION_MAJOR)
+	
+	# Apply plugin suport patch
+	patch -p1 < ../0001-Add-plugin-support-to-bash.patch
```

Comment:
>patch [](http://example.com/codeflow?start=1&length=5)

Use a series file so it will be easy to extend to many patch files.

Reply:
Fixed, use quilt to manage series patch files.